src/main.cc

```diff
@@ -2,20 +2,29 @@
 #include <memory>
 
 #include "GMSH.cc"
+#include "IO.h"
+#include "input.h"
 
 int main(int argc, char** argv) {
   try {
     if (argc != 3) {
       std::cout << "Usage: ./mpm-point-generator /path/to/mesh_file.msh\n"
-                   "Dimenssion 2 or 3\n";
+                   "Dimenssion 2 or 3 \n";
       throw std::runtime_error("Incorrect number of input arguments");
     }
-
+    //! Get dimension or Tdim
     int dimension = std::atoi(argv[2]);
-    std::string filename = argv[1];
 
+    //! Store input/output options including directory
+    std::string inputfile = argv[1];
+    std::string output_point = "/vertexcheck.txt" ;
```

Comment:
This is storing the file in the root directory. This will never work, permission issue.